cmd/ebrelayer/txs/parser.go

```diff
@@ -62,6 +79,58 @@ func LogLockToEthBridgeClaim(valAddr sdk.ValAddress, event *events.LockEvent) (e
 	return witnessClaim, nil
 }
 
+// ProphecyClaimToSignedOracleClaim : packages and signs a prophecy claim's data, returning a new oracle claim
+func ProphecyClaimToSignedOracleClaim(
```

Comment:
is this EthereumProphecyClaimToSignedCosmosOracleClaim ? or is it something else. should name it that if so, or add comment to explain